src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java

```diff
@@ -314,6 +312,40 @@ else if (exercise instanceof ModelingExercise) {
         return studentExams;
     }
 
+    /**
+     * Helper method to return a map for each user to their exercises
+     *
+     * @param studentExams the student exams of the users containing the exercises
+     * @return a map of the User as key, and a list of the users exercises as value
+     */
```

Comment:
I think you should mention in the documentation that quiz exercises will be excluded from the exam exercises (or generally that some exercises are filtered out)

Reply:
I adapted the docs accordingly.